### ISSUE TITLE
MathNet.Numerics.Signed NuGet Package Fails to Install on .NET Framework v3.5

### ISSUE THREAD

**Issue by jecaruthersjr** [mathnet/mathnet-numerics#449]
The MathNet.Numerics.Signed NuGet package fails to install when targeting .NET Framework 3.5.

Here is the output.

Attempting to gather dependency information for package 'MathNet.Numerics.Signed.3.13.1' with respect to project 'Test', targeting '.NETFramework,Version=v3.5'
Gathering dependency information took 217.09 ms
Attempting to resolve dependencies for package 'MathNet.Numerics.Signed.3.13.1' with DependencyBehavior 'Lowest'
Resolving dependency information took 0 ms
Resolving actions to install package 'MathNet.Numerics.Signed.3.13.1'
Resolved actions to install package 'MathNet.Numerics.Signed.3.13.1'
Retrieving package 'MathNet.Numerics.Signed 3.13.1' from 'nuget.org'.
Install failed. Rolling back...
Package 'MathNet.Numerics.Signed.3.13.1' does not exist in project 'Test'
Package 'MathNet.Numerics.Signed.3.13.1' does not exist in folder 'C:\Projects\Test\packages'
Executing nuget actions took 52.57 ms
Could not install package 'MathNet.Numerics.Signed 3.13.1'. You are trying to install this package into a project that targets '.NETFramework,Version=v3.5', but the package does not contain any assembly references or content files that are compatible with that framework. For more information, contact the package author.

The unsigned version works fine.

**Comment by cdrnet** [comment]
Yes, the signed package has always only included the .Net 4.0 build. Do you have a strict need for a signed build compatible to .Net 3.5?

**Comment by jecaruthersjr** [comment]
Yes, the library I am working on must stored in the GAC.  I know there are alternatives, but they will make things tough due to the fact this library is being built on a remote build server. Is there a way around this? Thanks.

**Comment by cdrnet** [comment]
The build script will have to be adapted, but it is certainly possible.

**Comment by jecaruthersjr** [comment]
Good to hear.  Would you be willing to alter the build script?
